src/visual/GameStateVisualizer.cs

```diff
@@ -1,12 +1,58 @@
 using System;
+using System.Linq;
+using NUnit.Framework;
+using System.Collections.Generic;
 
 namespace tcg
 {
   static class GameStateVisualizer
   {
-    static string Visualize(GameState state)
+    public static string OutputBoard(Player currPlayer){
```

Comment:
Better name is `BoardToString`
```suggestion
    public static string BoardToString(Player currPlayer){
```